Formula/clozure-cl.rb

```diff
@@ -36,7 +36,8 @@ def install
                          "-e", "(ccl:xload-level-0)",
                          "-e", "(ccl:compile-ccl)",
                          "-e", "(quit)"
-    system "echo \"(ccl:save-application \\\"dx86cl64.image\\\")\\n(quit)\" | ./dx86cl64 -n --image-name x86-boot64.image"
+    (testpath/"example").write('(ccl:save-application "dx86cl64.image")\n(quit)\n')
```

Comment:
`buildpath` instead. But why do we need to write it to a file and then `cat`?

Reply:
to fix rubocop issue